config/boot.rb

```diff
@@ -3,4 +3,5 @@
 unless File.exist?(File.expand_path('../../Gemfile.in', __FILE__))
   ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
   require 'bundler/setup' if File.exist?(ENV['BUNDLE_GEMFILE'])
+  RUBY_VERSION >= '2.3' && require('bootsnap/setup')
```

Comment:
I would still recommend wrap this rescue on load error: in production this would break on RUBY_VERIONS >= '2.3', also, I might want to run tests without `development` bundler group